force-app/main/default/classes/TestRunProcessor.cls

```diff
@@ -103,11 +103,40 @@ public with sharing class TestRunProcessor implements Schedulable {
                     Method_Pass__c = apexTestResult.Outcome == 'Pass' ? true : false,
                     Name = apexTestResult.ApexClass.Name,
                     Stack_Trace__c = apexTestResult.stackTrace,
-                    Test_Run__c = testRun.Id
+                    Test_Run__c = testRun.Id 
                 )
             );
+            
         }
-
-        insert results;
+        
+		List<Test_Run_Method_Result__c> result = processPassFailDates(results);
```

Comment:
looks like you've got some whitespace issues (tabs vs spaces).  Did you run `npm install` to get the prettier cli?

Reply:
fixed